### ISSUE TITLE
Installation process issues.

### ISSUE THREAD

**Issue by Radomirius** [yoshiask/FluentStore#13]
Hello Developer! Thank You for creating this app. I would want to draw attention to the following problem with the installer: there are 'bad behaviours' with the installation status reports and the installer itself - the Fluent Store has been installed but the program [Setup - Fluent Store Beta] wasn't closed automatically and just got stuck in some state.

Sorry for inaccurate English.
![2021-09-05 140030](https://user-images.githubusercontent.com/81755400/132124427-2c504de0-3725-446a-b4dc-7f425a89955e.png)

**Comment by yoshiask** [comment]
Hmm, that's not normal behavior at all. The garbled text leads me to believe that the translations included with the internal install script (which Microsoft created, not me) got corrupted somehow. Does this happen when you rerun the installer? If so, can you redownload the installer and try again?
Thanks!

**Comment by Radomirius** [comment]
(Windows 10, Release Preview channel build 19043.1202) I reran the installer - nothing's changed. I downloaded it and then initiated the installation - again, nothing's changed. I tried to reinstall the program on Win 11, build 22449.1000, - no any changes. 
If/when You release new version of Fluent Store, I will (probably) try running the installer once more, and (in case of detecting anything different) will notify You.
Perhaps, these problems occur because of Windows Insider builds or because of primary language, which is Russian.
